lib/files.ts

```diff
@@ -31,27 +31,53 @@ export function getCategoryDisplayName(filePath: string): string {
 export async function getDocsContent(docsDir: string): Promise<DocsFile[]> {
   const files = await fg(["**/*.mdx"], {
     cwd: docsDir,
-    ignore: ["reference/endpoints/**", "**/_*.mdx"],
+    ignore: ["**/_*.mdx"],
     absolute: true,
     dot: false,
   });
 
   const scanned = await Promise.all(
     files.map(async (file) => {
-      const fileContent = await fs.readFile(file);
-      const { content, data } = matter(fileContent.toString());
       const relativePath = path.relative(docsDir, file);
       const category = getCategoryDisplayName(relativePath);
-      const processed = await processContent(content);
+
+      // Check if this is an API reference page (not the index page)
+      const isApiReferencePage = relativePath.includes("reference/endpoints/") &&
```

Comment:
Do we wanna normalize this? Probably doesn't matter if its always running on vercel? `relativePath.split(path.sep).join("/")` would do it tho.